src/components/screens/monitor/network/flag.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import styles from './flag.css';
+
+const codes = {
+  dz: 'algeria',
+  am: 'armenia',
+  at: 'austria',
+  az: 'azerbaijan',
+  bs: 'bahamas',
+  bh: 'bahrain',
+  bd: 'bangladesh',
+  bb: 'barbados',
+  be: 'belgium',
+  bj: 'benin',
```

Comment:
Why not use the ISO 3166-1 alpha-2 two-letter codes already in the `src/components/screens/monitor/network/flag.css` file? I suppose the flag.css file is generated by some tool? I think there should be a tool out there to already use ISO 3166-1 alpha-2 as the class names. 

This would bring less complexity and more countries supported out-of-the-box.

Reply:
This was an auto-generated dictionary. Since it's an object and doesn't cause big-o overload, I didn't see the need to spend time on it.

Reply:
For the computational complexity, it's fine. :+1: 

What is the requirement of how many countries should we support?

Now, this file supports about 90 countries, but AFAIK there are almost 200 countries in the world and the flag.css seems to cover all of them.